keps/sig-network/1669-graceful-termination-local-external-traffic-policy/README.md

```diff
@@ -189,34 +194,25 @@ Try to be as paranoid as possible - e.g., what if some components will restart
 mid-rollout?
 -->
 
-TBD for beta.
+Roll out can fail if there are pods receiving traffic during termination but are unable to handle it.
+This can be unexpected, however, it is not any worse than existing behavior where kube-proxy would simply
+blackhole that traffic.
 
 ###### What specific metrics should inform a rollback?
 
-<!--
-What signals should users be paying attention to when the feature is young
-that might indicate a serious problem?
--->
-
-TBD for beta.
+Application-level metrics should be used to determine if traffic received during termination is causing issues.
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-<!--
-Describe manual testing that was done and the outcomes.
-Longer term, we may want to require automated upgrade/rollback tests, but we
-are missing a bunch of machinery and tooling and can't do that now.
--->
-
-TBD for beta.
+No, but upgrade testing should be done prior to Beta.
```

Comment:
Automated or just manual? [I guess manual - if so, let's make it explicit]
```
Manual tests are still to be run.
```

Also - please ensure that the results will be added here to the KEP before the actual graduation will happen in k/k code [something like: https://github.com/kubernetes/enhancements/pull/2538/files ]

Reply:
ack, will update. I think @aojea mentioend that openshift does have some automated testing around this, but I'm not sure we'll get an upstream signal for it.